qemu/tests/cfg/virtio_port_hotplug.cfg

```diff
@@ -44,6 +44,17 @@
             sleep_time = 0.5
             serials += " vs1"
             serial_type_vs1 = virtserialport
+        - re-use_chardev:
```

Comment:
1. case name contain both '-' and '_', better use '_'.
2. test case `unplug_port` reuse chardev too, so i think it is unreasonable this variant with same level.
It is better to use sub variants for `unplug_port`. Something like: `unplug_port.default`, `unplug_port.with_file_transfer/with_stress..`.

Reply:
update and thanks.